filters/encrypt/docs_test.go

```diff
@@ -0,0 +1,109 @@
+package encrypt_test
+
+import (
+	"context"
+	"crypto/rand"
+	"encoding/base64"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/hashicorp/eventlogger"
+	"github.com/hashicorp/eventlogger/filters/encrypt"
+	"github.com/hashicorp/eventlogger/sinks/writer"
+	wrapping "github.com/hashicorp/go-kms-wrapping"
+	"github.com/hashicorp/go-kms-wrapping/wrappers/aead"
+)
+
+func ExampleFilter() {
+	then := time.Date(
+		2009, 11, 17, 20, 34, 58, 651387237, time.UTC)
+	// Create a broker
+	b := eventlogger.NewBroker()
+
+	b.StopTimeAt(then) // setting this so the output timestamps are predictable for testing.
+
+	wrapper := exampleWrapper()
+
+	// A gated.Filter for events
+	f := &encrypt.Filter{
+		Wrapper:  wrapper,
+		HmacSalt: []byte("salt"),
+		HmacInfo: []byte("info"),
+	}
+	// Marshal to JSON
+	jsonFmt := &eventlogger.JSONFormatter{}
+
+	// Send the output to stdout
+	stdoutSink := &writer.Sink{
+		Writer: os.Stdout,
+	}
+
+	// Register the nodes with the broker
+	nodes := []eventlogger.Node{f, jsonFmt, stdoutSink}
+	nodeIDs := make([]eventlogger.NodeID, len(nodes))
+	for i, node := range nodes {
+		id := eventlogger.NodeID(fmt.Sprintf("node-%d", i))
+		err := b.RegisterNode(id, node)
+		if err != nil {
+			// handle error
+		}
+		nodeIDs[i] = id
+	}
+
+	et := eventlogger.EventType("test-event")
+	// Register a pipeline for our event type
+	err := b.RegisterPipeline(eventlogger.Pipeline{
+		EventType:  et,
+		PipelineID: "encrypt-filter-pipeline",
+		NodeIDs:    nodeIDs,
+	})
+	if err != nil {
+		panic(err)
+	}
+
+	p := &struct {
+		NoClassification string
+		Public           string `classified:"public"`
+		Sensitive        string `classified:"sensitive,redact"`
```

Comment:
I think it'd be good to put an example of this style ("sensitive,redact") in the docs so it's clear you can override via the tags

Reply:
done.